src/java/textdisplay/Project.java

```diff
@@ -606,6 +606,45 @@ public Date getModification() throws SQLException {
             closePreparedStatement(ps);
         }
     }
+    
+    /**
+     * Get all Projects that are connected with the DLA projects.
+     * They should all have titles like 
+     *
+     * @return
+     * @throws java.sql.SQLException
+     */
+    public static Project[] getAllDunbarProjects() throws SQLException {
+        String query = "select distinct(project.id) from project where "
+                + "project.name like 'F-%' or "
```

Comment:
If they are like `F-123-12 word word` we can do `F-_%-_% _` (F followed by hyphen something followed by hyphen something followed by nbsp; something)

Reply:
I think that is likely. The default title is shorter, however. I think you won't get (m)any bad results with just `F[0-9][0-9][0-9]`. The dash is likely, but maybe not promised.

Reply:
I'm actually starting to think that this may be much easier to do from an iframe instead. Also, see the mention I gave for possibly approaching this from the other end.